File: uwp/ArtistWayUWP/Models/WeekContent.cs
using System.Collections.Generic;

namespace ArtistWayUWP.Models
{
    public sealed class WeekContent
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Intro { get; set; }
        public List<string> Essay { get; set; } = new List<string>();
        public List<ChecklistItem> Checklist { get; set; } = new List<ChecklistItem>();
        public string CheckinBonus { get; set; }
    }
}

File: android/ArtistWayAndroid/app/src/main/java/com/rodcarvalho/artistway/data/model/WeekContent.kt
package com.rodcarvalho.artistway.data.model

import kotlinx.serialization.Serializable

// "list" -> NamedListScreen(listKey); "screen" -> uma das poucas telas
// fixas de ferramenta (lifePie, circuloSeguranca, principiosBasicos).
@Serializable
data class ChecklistLink(
    val type: String = "",
    val key: String = "",
)

@Serializable
data class ChecklistItem(
    val task: String = "",
    val detail: String = "",
    val link: ChecklistLink? = null,
)

@Serializable
data class WeekContent(
    val id: Int,
    val title: String = "",
    val intro: String = "",
    val essay: List<String> = emptyList(),
    val checklist: List<ChecklistItem> = emptyList(),
    val checkinBonus: String = "",
)
